src/nxdoc/nuxeo-server/administration/database-configuration.md

```diff
@@ -24,145 +21,18 @@ version_override:
-        date: '2014-05-27 15:29'
-        message: ''
-        version: '16'
-    -
-        author: Solen Guitter
-        date: '2013-10-21 16:43'
-        message: Updated PostgreSQL version
-        version: '15'
-    -
-        author: Solen Guitter
-        date: '2012-03-28 18:29'
-        message: ''
-        version: '14'
-    -
-        author: Solen Guitter
-        date: '2012-03-28 18:29'
-        message: ''
-        version: '13'
-    -
-        author: Solen Guitter
-        date: '2012-03-28 18:29'
-        message: ''
-        version: '12'
-    -
-        author: Solen Guitter
-        date: '2011-03-11 11:19'
-        message: ''
-        version: '11'
-    -
-        author: Solen Guitter
-        date: '2011-03-11 10:55'
-        message: ''
-        version: '10'
-    -
-        author: Solen Guitter
-        date: '2011-03-04 17:32'
-        message: ''
-        version: '9'
-    -
-        author: Solen Guitter
-        date: '2011-03-04 17:23'
-        message: Reverted from v. 6
-        version: '8'
-    -
-        author: Solen Guitter
-        date: '2011-03-04 17:23'
-        message: ''
-        version: '7'
-    -
-        author: Solen Guitter
-        date: '2011-03-02 18:11'
-        message: ''
-        version: '6'
-    -
-        author: Solen Guitter
-        date: '2011-03-02 15:01'
-        message: ''
-        version: '5'
-    -
-        author: Florent Guillaume
-        date: '2010-12-28 14:59'
-        message: ''
-        version: '4'
-    -
-        author: Stéfane Fermigier
-        date: '2010-12-27 10:24'
-        message: ''
-        version: '3'
-    -
-        author: Florent Guillaume
-        date: '2010-06-17 16:52'
-        message: ''
-        version: '2'
-    -
-        author: Florent Guillaume
-        date: '2010-06-17 16:47'
-        message: ''
-        version: '1'
-
 ---
-Nuxeo applications store most of their data in a SQL database. Several databases are supported, but they must be configured to work correctly.
+Nuxeo applications store most of their data in a database, that can be either a NoSQL database such as MongoDB or an object-relational database such as PostgreSQL. Several databases are supported, but they must be configured to work correctly.
 
 This takes two steps:
 
-1.  Configure the database:
-    *   [PostgreSQL ]({{page page='postgresql'}})
-    *   [Oracle ]({{page page='oracle'}})
-    *   [Microsoft SQL Server ]({{page page='microsoft-sql-server'}})
-    *   [MySQL]({{page page='mysql'}})
-    *   [MariaDB]({{page page='mariadb'}})
-    *   [MongoDB]({{page page='mongodb'}})
-2.  [Connect Nuxeo to the database]({{page page='connecting-nuxeo-to-the-database'}}).
+1.Configure the database:
+
+* [MongoDB]({{page page='mongodb'}})
+* [PostgreSQL ]({{page page='postgresql'}})
+* [MySQL]({{page page='mysql'}})
+* [MariaDB]({{page page='mariadb'}})
+* [Oracle ]({{page page='oracle'}})
+* [Microsoft SQL Server ]({{page page='microsoft-sql-server'}})
```

Comment:
nitpick:
```suggestion
* [MongoDB]({{page page='mongodb'}})
* [PostgreSQL ]({{page page='postgresql'}})
* [Oracle ]({{page page='oracle'}})
* [Microsoft SQL Server ]({{page page='microsoft-sql-server'}})
* [MySQL]({{page page='mysql'}})
* [MariaDB]({{page page='mariadb'}})
```
to conserve the original order modulo MongoDB + PostgreSQL in first position.

Reply:
Looks like this is sorted alphabetically with _SQL Server_ taken into account instead of _Microsoft SQL Server_. We can keep it as it.

Reply:
It was sorted by "preferred" database, but yes we can keep the old one or use alphabetical sort. What do we chose ? :smile:

Reply:
I like the "by preferred" DB way!